src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -1739,7 +1739,7 @@ public async Task SaveAsync()
 
                     var indexCount = _command.Static.Count + _command.Auto.Count;
                     var operationTimeout = _store._serverStore.Engine.OperationTimeout;
-                    var timeout = TimeSpan.FromSeconds((indexCount / 50.0) * operationTimeout.TotalSeconds);
+                    var timeout = TimeSpan.FromSeconds((indexCount / 10.0) * operationTimeout.TotalSeconds);
```

Comment:
It needs to take into account the number of cores on that machine

Reply:
There is no good value here. My machine can have 12 cores but different one in the cluster have have 1.